### PR TITLE
Fix: ActivityReactionsDrawer: Make sure to display user popover while hovering on user names.

### PR DESCRIPTION
**Pull request by mdiha**
I added the user popover to be displayed while hovering on user names in activity reactions drawer.

### CODE REVIEW

**Review by ahamdi — COMMENTED** [review]
Could you add more details about the problem and how will it be fixed by this PR.

File: webapp/portlet/src/main/webapp/activity-reactions/components/ActivityReactionsListItems.vue [hunk 1]
```diff
@@ -53,6 +55,9 @@ export default {
   },
   data () {
     return {
+      cmpId: `chip${parseInt(Math.random() * 10000)
+        .toString()
+        .toString()}`,
```
Comment (ahamdi) [inline webapp/portlet/src/main/webapp/activity-reactions/components/ActivityReactionsListItems.vue:59]: Is there a reason to have double toString() ?

File: webapp/portlet/src/main/webapp/activity-reactions/components/ActivityReactionsListItems.vue [hunk 2]
```diff
@@ -53,6 +55,9 @@ export default {
   },
   data () {
     return {
+      cmpId: `chip${parseInt(Math.random() * 10000)
```
Comment (ahamdi) [inline webapp/portlet/src/main/webapp/activity-reactions/components/ActivityReactionsListItems.vue]: What does the word Chip refer to ?

File: webapp/portlet/src/main/webapp/activity-reactions/components/ActivityReactionsListItems.vue [hunk 3]
```diff
@@ -89,6 +95,18 @@ export default {
           console.error('Error processing action', e);
         });
     },
+    initTiptip() {
+      this.$nextTick(() => {
+        $(`#${this.cmpId}`).userPopup({
+          restURL: '/portal/rest/social/people/getPeopleInfo/{0}.json',
```
Comment (ahamdi) [inline webapp/portlet/src/main/webapp/activity-reactions/components/ActivityReactionsListItems.vue:108]: Please check if you do not need more initiation attributes ? labels for example

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: webapp/portlet/src/main/webapp/activity-reactions/components/ActivityReactionsListItems.vue (hunk 3 above)
Comment (mdiha) [reply webapp/portlet/src/main/webapp/activity-reactions/components/ActivityReactionsListItems.vue:108]: Okay

File: webapp/portlet/src/main/webapp/activity-reactions/components/ActivityReactionsListItems.vue (hunk 2 above)
Comment (mdiha) [reply webapp/portlet/src/main/webapp/activity-reactions/components/ActivityReactionsListItems.vue]: My bad i will change it

File: webapp/portlet/src/main/webapp/activity-reactions/components/ActivityReactionsListItems.vue (hunk 1 above)
Comment (mdiha) [reply webapp/portlet/src/main/webapp/activity-reactions/components/ActivityReactionsListItems.vue:59]: No i will fix it

**Review by ahamdi — APPROVED** [review]
LGTM

**Pull request merged**